frontend/.eslintrc

```diff
@@ -70,7 +72,28 @@
     // It's OK to use ts-ignore when we're missing type definitions
     // for a JS library.
     "@typescript-eslint/ban-ts-ignore": "off",
-    "@typescript-eslint/ban-ts-comment": "off"
+    "@typescript-eslint/ban-ts-comment": "off",
+    // Permit for-of loops (https://stackoverflow.com/a/42237667)
+    "no-restricted-syntax": [
+      "error",
+      "ForInStatement",
+      "LabeledStatement",
+      "WithStatement"
+    ],
+    // It's okay to use parseInt() with the default radix of 10
+    "radix": "off",
+    // Disable a bunch of AirBNB rules we're currently in violation of.
+    // TODO: For each one, either fix and reenable, or provide a justification.
+    "import/extensions": "off",
+    "import/prefer-default-export": "off",
+    "import/no-extraneous-dependencies": "off",
+    "max-classes-per-file": "off",
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "no-plusplus": "off",
+    "no-console": "off",
```

Comment:
I think we should keep the no-console on. We have our log.ts that everything should theoretically be using. We could just disable the rule within that file

Reply:
Good point! I've turned on `no-console` everywhere but in our custom log.ts, as suggested.